packages/menu/test/submenu.test.ts

```diff
@@ -1068,4 +1068,65 @@ describe('Submenu', () => {
         expect(rootItem1.open, 'finally closed 1').to.be.false;
         expect(rootItem2.open, 'finally closed 2').to.be.false;
     });
+    it('supports using non-menu-item elements as the root of a submenu', async () => {
```

Comment:
nit: "supports" sounds like all aspects of the component work (selection, keyboard navigation,...).